README.md

```diff
@@ -37,13 +37,26 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
 
 ## Installation
 
+### Cocoapods
+Use following pods in your `Podfile` to install FR iOS SDK module(s) you want to install the latest version.
+
 ```
 pod 'FRAuth' // Authentication module for Access Manager
 pod 'FRUI' // UI SDK demonstrates FRAuth SDK functionalities
 pod 'FRProximity' // Add-on for FRAuth's Device Profile feature related to location, and BLE
 pod 'FRAutheneticator' // Authenticator module that generates HOTP, TOTP, and Push registration/authentication
+pod 'FRGoogleSignIn' // Social Login module for Google Sign In
+pod 'FRFacebookSignIn' // Social Login module for Facebook Login
 ```
 
+### Swift Package Manager
+> The ForgeRock iOS SDK is only available via Swift Package Manager for 3.0.0 and above. Any versions 2.2.0 and below are only available via Cocoapods
```

Comment:
I would rephrase that to something like:
The ForgeRock iOS SDK is also available via the Swift Package Manager from version 3.0.0 and above. 
The 'FRGoogleSignIn' module is not available via the Swift Package Manager.
Any older versions (2.0.0 and below) are only available via Cocoapods.

Reply:
I've updated the text blocks adding more information. Please let me know if you have any further concerns.